packaging/hudi-cli-bundle/pom.xml

```diff
@@ -0,0 +1,275 @@
+                                    <resource>META-INF/LICENSE</resource>
+                                    <file>target/classes/META-INF/LICENSE</file>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>META-INF/spring.handlers</resource>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>META-INF/spring.schemas</resource>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>META-INF/spring.factories</resource>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                    <mainClass>org.apache.hudi.cli.Main</mainClass>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>META-INF/spring/org.springframework.boot.actuate.autoconfigure.web.ManagementContextConfiguration.imports</resource>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                    <resource>META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports</resource>
+                                </transformer>
+                            </transformers>
+                            <artifactSet>
+                                <includes combine.children="append">
+                                    <include>org.apache.hudi:hudi-cli</include>
+                                    <include>org.apache.hudi:hudi-utilities_${scala.binary.version}</include>
+
+                                    <!-- cli related dependencies -->
+                                    <include>org.springframework*:*</include>
+                                    <include>org.springframework.boot:*</include>
+                                    <include>org.apache.logging.log4j:*</include>
+                                    <include>org.scala-lang:scala-library</include>
+                                    <include>org.apache.parquet:parquet-hadoop</include>
+                                    <include>com.google.guava:guava</include>
```

Comment:
`com.google.guava` should be avoided or at least shaded.

Reply:
Could you check hudi-presto-bundle or hudi-trino-bundle for relevant relocation rules?  The hudi-cli-bundle might need those rules as well.